relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -24,6 +24,18 @@ pub mod types;
 /// cardinality data such as raw URLs.
 const PLACEHOLDER_UNPARAMETERIZED: &str = "<< unparameterized >>";
 
+/// Tags we set on metrics for performance score measurements (e.g. `score.lcp.weight`).
+///
+/// These are a subset of "universal" tags.
+const PERFORMANCE_SCORE_TAGS: [CommonTag; 6] = [
+    CommonTag::Release,
+    CommonTag::Environment,
+    CommonTag::Transaction,
+    CommonTag::TransactionOp,
+    CommonTag::BrowserName,
+    CommonTag::TransactionOp,
```

Comment:
Seems like TransactionOp appears here twice, is it intentional? Also, I think we may want to include GeoCountryCode in case we may need to facet by country code in the future

Reply:
Whoops, looks like I forgot to replace that second one with country code. Fixed now, thanks for catching it!